src/controllers/controlpickermenu.cpp

```diff
@@ -272,14 +272,10 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
             tr("Adjust Beatgrid - Match Alignment"),
             tr("Adjust beatgrid to match another playing deck."),
             pBpmMenu);
-    addDeckAndSamplerControl("lock_tempo",
+    addDeckAndSamplerControl("toggle_lock_bpm",
```

Comment:
Please use `bpm_toggle_lock` so we have 'hierarchical' naming structure

Reply:
Yeah that's a "git moment", as it's actually good on my computer and I have nothing to commit nor push :-) 

I'll see that in 3h.

Reply:
(Done)